### PR TITLE
Standards: adding a bunch of standards taken from the ESO database

### PR DESCRIPTION
**Pull request by ntejos**
Follow-up on previous PR#1202, I have added now a bunch of of standards taken from the ESO database
I followed the instructions as given in  https://pypeit.readthedocs.io/en/release/fluxing.html#adding-a-standard-star

Importantly, a couple of files in the `esofil/` directory have been commented out from the `esofil_info.txt` because these were in magnitude units rather than flux units (as expected by `core.flux_calib`). Paging @Tang-SL (who introduced these in PR #1180  for checking if is ok to comment these out.)

I've also tweaked a bit the documentation for adding-a-standard-star

### CODE REVIEW

**Comment by profxj** [comment]
@ntejos -- Looks like you have a conflict with `develop`

Can you pull that in and resolve before I review?  Thanks.

**Comment by ntejos** [comment]
@profxj - Sure! I think the conflict has been resolved now.

**Comment by Tang-SL** [comment]
@ntejos Thanks for this PR, I think you are right, the eso .dat file can only be in flux format. Though such a sentence exists in the aaareadme.ctio file: The 'm' files list wavelength ( A ), AB magnitude and bin (A). The file name consists of a prefix 'm' and the star name. 
That's why I put an 'm' file for those two stars there. I have tested both, only 'f' format works properly.

**Comment by ntejos** [comment]
I have added an assertion to make sure ESO files used, always start with name 'f'. This will prevent anyone mistakenly using a file calibrated in magnitudes (rather than in flux) retrieved from the ESO database.

**Review by profxj — APPROVED** [review]
Great stuff

File: doc/fluxing.rst [hunk 1]
```diff
@@ -222,8 +222,8 @@ If your star is not in the repository you can add in a new
 solution if it is in the
 `ESO database <https://www.eso.org/sci/observing/tools/standards/spectra/stanlis.html>`_.
 
-You will need to place their .dat file in pypeit/data/standards/esofil/
-and then edit the *esofil_info.txt* file in their accordingly.
+You will need to place their .dat file (make sure the flux column is in flux units rather than magnitudes, i.e. those files starting with `f`)
```
Comment (jhennawi) [inline doc/fluxing.rst]: Can you please add the flux units of the files to these docs?

**Review by jhennawi — APPROVED** [review]
Thanks for the PR @ntejos. Minor document change requested, otherwise you are good to go.

**Pull request merged**